salesagility/SuiteCRM
# Fixing compose email link in activities subpanel in opportunities

issue: #5639

## Description



Fixing compose email link in activities subpanel in opportunities

## Motivation and Context

- Fixes the feature in the opportunities

## How To Test This

- Create an opportunity or view an existing opportunity
- In the detail view expand the activities sub panel
- Select the drop down menu inside toolbar of the subpanel
- You should see compose email link

## Types of changes


### Final checklist
